github_bin.js

```diff
@@ -0,0 +1 @@
+var GITHUB_TOKEN = '575a5ec38cb1397e61eccb8347fda2f657f56060'
```

Comment:
Definitely want to get rid of this... I would suggest removing this token from your Personal Access Tokens if you've not already done so. You can create another, but be sure not to share it.